scripts/gofmt.sh

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env bash
```

Comment:
Please rename to `check-gofmt.sh` for consistency with other check scripts and remove the conditions `ONLY_GIT_STAGED` (see below) and `ABORT_ON_ERR` (should always fail on errors).

If we need a separate way to format Go code besides calling `go fmt <filename>`, there should be a make target like `make format` or similar (not needed for now).